roles/upgrade/post-upgrade/tasks/main.yml

```diff
@@ -4,3 +4,8 @@
   delegate_to: "{{ groups['kube-master'][0] }}"
   when:
     - needs_cordoning|default(false)
+
+- name: Reload kubelet
```

Comment:
Upgrading/deploying with cluster.yml should work too. 
The reload kubelet could be an handler when the manifests or the kubelet verison (systemd) file is changed.

Reply:
yeah,  it will triggered to restart kubelet when use pgrading/deploying with cluster.yml 
I am sure, manifests or the kubelet verison (systemd) file is changed when upgrade k8s.
so add handler in here?

Reply:
node :https://github.com/kubernetes-incubator/kubespray/blob/master/roles/kubernetes/node/handlers/main.yml
master:https://github.com/kubernetes-incubator/kubespray/blob/master/roles/kubernetes/master/handlers/main.yml

Reply:
why they are not triggered when using the upgrade playbook?

Reply:
There are problem with the judgment when execute the upgrade.  So we have to find it.